outlook/src/classes/Company.ts

```diff
@@ -33,10 +42,27 @@ class Company {
         this.additionalInfo = {};
     }
 
-    static fromJSON(o: Object): Company {
+    static fromJSON(o: Object, enrichmentInfo?: EnrichmentInfo): Company {
         if (!o) return new Company();
         const company = Object.assign(new Company(), o);
         company.address = Address.fromJSON(o['address']);
+
+        if (enrichmentInfo != null) {
+            if (enrichmentInfo.type == EnrichmentInfoType.NoData) {
+                company.enrichmentStatus = EnrichmentStatus.enrichmentEmpty;
+            } else {
+                if (EnrichmentInfoType.CompanyCreated || EnrichmentInfoType.CompanyUpdated) {
+                    company.enrichmentStatus = EnrichmentStatus.enriched;
```

Comment:
Don't we need and/or won't we use difference between created and updated ? Not sure, if we update some UI for example. Or maybe we simply use EnrichmentInfoType ?

Reply:
The UI is updated slightly differently if a company is updated, compared to when it's created